pages/donate.html

```diff
@@ -1,7 +1,14 @@
 <section class="inner">
-  <h2>Donate</h2>
 
-  <p>
-    Placeholder content for donate page.
-  </p>
+	<h2>Donate to The Oppia Foundation</h2>
```

Comment:
Throughout, please use spaces not tabs. (There should be a suitable setting in your IDE/editor that enforces this.)

Reply:
done